<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepSearch</class>
 <widget class="QMainWindow" name="DeepSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deep Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:0 15px;</string>
        </property>
        <property name="text">
         <string>Searching Table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databaseOptions">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0 15px;</string>
        </property>
        <item>
         <property name="text">
          <string>Book Registry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Student Registry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Borrow Ledger</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Search Based on</string>
      </property>
      <widget class="QRadioButton" name="titleRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>112</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="authorRadio">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>112</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="publisherRadio">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>241</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Publisher</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="bookidRadio">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>40</y>
         <width>112</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Book ID</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchingArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitBtn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:0 10px;
font-size:15px;</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
